test/test_utils.sh

```diff
@@ -50,6 +50,36 @@ build_and_push_fetcher() {
     popd
 }
 
+build_and_push_python_env_runtime() {
+    image_tag=$1
+
+    pushd $ROOT/environments/python3/
+    docker build -t $image_tag .
+
+    gcloud_login
+    
+    gcloud docker -- push $image_tag
+    popd
+}
+
+build_and_push_python_env_builder() {
+    image_tag=$1
+
+    pushd $ROOT/builder/cmd
```

Comment:
Can you separate this into a different function? Builder is not python specific and shouldn't be built inside python build scripts